src/plone/api/content.py

```diff
@@ -16,18 +21,24 @@
 from zope.interface import Interface
 from zope.interface import providedBy
 
-import pkg_resources
 import random
 import transaction
 
 try:
-    pkg_resources.get_distribution('Products.Archetypes')
-except pkg_resources.DistributionNotFound:
+    get_distribution('Products.Archetypes')
+except DistributionNotFound:
     class IBaseObject(Interface):
```

Comment:
I did not add this but it is a good question. When Products.Archetypes is not installed  ``IBaseObject.providedBy(obj)`` would then always be true, would it not?
@jaroel Can you have a look at that (since you added it...)?